src/status_im/ui/screens/signing/views.cljs

```diff
@@ -84,10 +84,10 @@
        [{:style {:color colors/black}} (displayed-name contact)]]
       [react/text {:style {:margin-top 6 :color colors/gray}}
        (str fee " " fee-display-symbol " " (string/lower-case (i18n/label :t/network-fee)))])]
-   [button/button (merge {:type :secondary
-                          :container-style {:padding-horizontal 24}
-                          :label (i18n/label :t/cancel)}
-                         (when-not in-progress? {:on-press #(re-frame/dispatch [:signing.ui/cancel-is-pressed])}))]])
+   [button/button {:type :secondary
+                   :container-style {:padding-horizontal 24}
+                   :label (i18n/label :t/cancel)
+                   :on-press #(re-frame/dispatch [:signing.ui/cancel-is-pressed])}]])
```

Comment:
Notice that transactions can't actually be cancelled, so hitting `Cancel` while the tx is in-flight really just discards the popup. Might be worth considering a different label depending on context.